README.md

```diff
@@ -18,9 +18,43 @@
 ## Usage
 
 1. Choose your flavour.
-2. Copy the contents of _flavour_.cava into your Cava config file (usually located at: `$HOME/.config/cava/`), replacing the existing gradient settings.
+2. Copy the contents of _flavour_.cava into your Cava config file, as specified in [Cava's README](https://github.com/karlstav/cava#configuration). Typically, the config file is located at: `~/.config/cava/`. Ensure you replace the existing gradient settings.
+
+> [!WARNING]
+> Running any of the commands below may overwrite your existing cava configuration file.
+
```

Comment:
Would it be better to provide a cURL example instead? Is it more commonly installed OOTB than wget?